terraform/outputs.tf

```diff
@@ -85,3 +85,28 @@ output "bigquery_pubsub_destination" {
   description = "BigQuery PubSub destination"
   value       = format("${google_bigquery_table.events_table.project}:${google_bigquery_table.events_table.dataset_id}.${google_bigquery_table.events_table.table_id}")
 }
+
+output "pr_stats_looker_studio_report_link" {
+  description = "The Looker Studio Linking API link for connecting the data sources for the PR Stats dashboard."
+  value = var.pr_stats_dashboard.enabled ? join("",
+    [
+      "https://lookerstudio.google.com/reporting/create",
+      "?c.reportId=${var.pr_stats_dashboard.looker_report_id}",
+      "&r.reportName=PR%20Stats",
```

Comment:
@gjonathanhong Should the name be something more generic like GitHub Metrics or something if we decide to add more "base" reports?